Added calendar of related events.

bridgetkromhout opened the pull request:
I keep seeing discussions where people are trying to pick dates for their devopsdays and not being aware of other events in the same space which might affect their date selection. I'm attempting to alleviate that with this calendar (and I picked the URL to make it easy to remember/hand out).

Since this is a change to the site itself, it would be nice to have another core organizer review it (for concept/placement/idea - I've tested it technically).

![screen shot 2015-02-10 at 11 21 09 pm](https://cloud.githubusercontent.com/assets/2104453/6142425/e60eac96-b17b-11e4-8c07-3f7314c79919.png)

LanyonM commented:
fwiw, I think this change looks good and would be really helpful for local organizers  as we pick dates.  My only suggestion would be to add a link to it from the Organizing page (specifically http://www.devopsdays.org/pages/organizing/#6) so that it's easier to find.  I'd be happy to do that after the PR is merged.  :smile:

bridgetkromhout commented:
Answering @jedi4ever : 

> maybe have it come from the config yml from each event?

I'm embedding a google calendar with a lot of other stuff on it too (that doesn't have devopsdays.org events), so I'm not going to try to automagically populate it in the first pass. It would be cool if someone wanted to, though.

> Does it make sense to name it devopsdays events ? or are we going to be a calender for all #devops related events?

I changed the titles and wording on the pages to make it clear that devopsdays is going to also point to other events (principally to give our event organizers and attendees more context). 

If people do not want to point to other events from our events page, I can move this so it's only in the organizing section. I think it will be useful for organizers and attendees alike, but I am not strongly attached to having it in this specific place on the site. (It just seemed the most intuitive.) Would love to hear opinions from anyone who cares.

While I was at it, I fixed our events page. The manually-collected list of past events was a pain because very often, it would not get updated. Now we'll just have a single source of truth on past events.

> Maybe loose the ctz=America/Chicago in the URL ?

Google calendar requires a time zone. I changed it to London; at least that's UTC.

> who's owner of the calender? someone in the devopsdays.org google domain?

That's a much better idea than my first pass at this (which was my personal gmail account), so now it's hosted under the admin@devopsdays.org account.

Answering @LanyonM :  

> My only suggestion would be to add a link to it from the Organizing page 

The entire organizing section (among other sections of the site) needs revising, but yes. I just want to get this up so I have a canonical place to show people what their conflicts and constraints will be when they're picking their dates.

![screen shot 2015-02-16 at 6 37 41 pm](https://cloud.githubusercontent.com/assets/2104453/6221715/abfe3fae-b60c-11e4-82f3-cb6ab5d183a5.png)

![screen shot 2015-02-16 at 6 38 29 pm](https://cloud.githubusercontent.com/assets/2104453/6221717/aebe3e60-b60c-11e4-83d8-2e060ce300ee.png)

bridgetkromhout commented:
Okay, nobody else has weighed in on this. I'm going to merge it tomorrow if I hear no objections.

bridgetkromhout commented:
Merging this in. If you want to add events to the calendar, it's under the admin@devopsdays.org google account.
